frontend/views/pages/GroupDashboard.vue

```diff
@@ -126,6 +130,11 @@ export default ({
     },
     updateBannerVisibility () {
       this.ephemeral.hideBanner = localStorage.getItem(this.bannerStorageKey) === 'true'
+    },
+    async requestPermissions () {
+      await requestNotificationPermission()
```

Comment:
Nice catch. Thanks. @snowteamer

Reply:
What do you think if we do `removeEventListener` in `beforeDestroy` lifecycle?

Reply:
But `requestNotificationPermission` function can not throw any errors.
I think try/catch/finally isn't needed here.

Reply:
@Silver-IT But @snowteamer raises a good point - at some point in the future someone else could rewrite your code for `requestNotificationPermission` so that it does throw an exception. In that case this part of the app will break in a serious way.

Reply:
Since the current `requestNotificationPermission` function can not throw any errors, should we leave this without any changes? I think I should add comment here but to be honest, I don't think I should use try/finally at the moment.

Reply:
And in this case, if there's an exception, I believe @snowteamer's suggestion is to remove the event listener so that it doesn't get stuck in a loop. That sounds like a reasonable/wise course of action to me.

Reply:
Understand. Thanks, @taoeffect , @snowteamer.